### PR TITLE
export firstDay, dayNames and monthNames

### PR DESCRIPTION
**Pull request by georgehrke**
Is the `declare var firstDay:` ... thing right?
(Never used TypeScript before 🙈)

### CODE REVIEW

File: lib/index.ts [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 type OC16to17 = Nextcloud.v16.OC | Nextcloud.v17.OC
 declare var OC: OC16to17;
+declare var firstDay: number;
+declare var dayNames: String[];
+declare var dayNamesShort: String[];
+declare var dayNamesMin: String[];
+declare var monthNames: String[];
+declare var monthNamesShort: String[];
```
[the remark this answers is not in the document]
Comment (ChristophWurst) [reply lib/index.ts]: I can do that part for you if Typescript is too scare right now :)

[the remark this answers is not in the document]
Comment (georgehrke) [reply lib/index.ts]: Do i just have to put it into every OC.d.ts or is there a convenient way to include it like you did with OC, OCP and humanFileSize?

Or just feel free to do it :P

[the remark this answers is not in the document]
Comment (ChristophWurst) [reply lib/index.ts]: Let me have a look :)

[the remark this answers is not in the document]
Comment (ChristophWurst) [reply lib/index.ts]: Mind giving the fixup a test together with https://github.com/nextcloud/nextcloud-typings/pull/9? I didn't actually test this :)

[the remark this answers is not in the document]
Comment (ChristophWurst) [reply lib/index.ts]: Looking at the generated js it should be fine, though

**Comment by rullzer** [comment]
> > https://github.com/nextcloud/server/blob/f08aed74d78cc2ec50dc74154c2862d054791882/lib/private/Template/JSConfigHelper.php#L178
> 
> Why did I ask see_no_evil cry

the most beautiful code right! ;)

**Pull request merged**